shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -884,15 +885,16 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
           val pages = parquetFileReader.readNextRowGroup
           if (pages == null) {
             throw new IOException("expecting more rows but reached last" +
-              " block. Read " + rowsReturned + " out of " + totalRowCount)
+                " block. Read " + rowsReturned + " out of " + totalRowCount)
           }
           columnReaders = new Array[VectorizedColumnReader](columns.size)
           for (i <- 0 until columns.size) {
             if (!missingColumns(i)) {
               columnReaders(i) =
-                new VectorizedColumnReader(columns.get(i), types.get(i)
-                  .getOriginalType, pages.getPageReader(columns.get(i)), null /*convertTz*/,
-                  LegacyBehaviorPolicy.CORRECTED.toString, LegacyBehaviorPolicy.EXCEPTION.toString)
+                  new VectorizedColumnReader(columns.get(i), types.get(i)
+                      .getOriginalType, pages.getPageReader(columns.get(i)), null /*convertTz*/ ,
+                    LegacyBehaviorPolicy.CORRECTED.toString,
+                    LegacyBehaviorPolicy.EXCEPTION.toString)
```

Comment:
If we're already wrapping on multiple lines for the number of arguments, I don't think it's worth splitting an argument across lines that can fit on a line.  Personally I'd put one arg per line to make it easier to line up the arguments (both in terms of indentation and in terms of parameter ordering), e.g.:
```suggestion
                  new VectorizedColumnReader(
                    columns.get(i),
                    types.get(i).getOriginalType,
                    pages.getPageReader(columns.get(i)),
                    null /*convertTz*/ ,
                    LegacyBehaviorPolicy.CORRECTED.toString,
                    LegacyBehaviorPolicy.EXCEPTION.toString)
```